### ISSUE TITLE
[WARNING] Switching to AlarmControlPanelState instead of setting state directly.

### ISSUE THREAD

**Issue by r0bb10** [petrleocompel/hikaxpro_hacs#137]
```
Entity None (<class 'custom_components.hikvision_axpro.alarm_control_panel.HikAxProPanel'>) is setting state directly which will stop working in HA Core 2025.11. Entities should implement the 'alarm_state' property and return its state using the AlarmControlPanelState enum, please report it to the custom integration author.
```

Also reported after 2024.11 update.

**Comment by petrleocompel** [comment]
Will be fixed by #138

**Comment by petrleocompel** [comment]
Released as [v3.0.0-beta2](https://github.com/petrleocompel/hikaxpro_hacs/releases/tag/v3.0.0-beta2)

Later it will be upstreamed to "master" and pushed to "stable" release of 3.0.0.

**Comment by r0bb10** [comment]
@petrleocompel this is the new error report from hass after updating to 3b2

```
Detected that custom integration 'hikvision_axpro' is setting state directly. Entity None (<class 'custom_components.hikvision_axpro.alarm_control_panel.HikAxProPanel'>) should implement the 'alarm_state' property and return its state using the AlarmControlPanelState enum at custom_components/hikvision_axpro/alarm_control_panel.py, line 44: async_add_entities(panels, False). This will stop working in Home Assistant 2025.11, please create a bug report at https://github.com/petrleocompel/hikaxpro_hacs/issues

**Comment by petrleocompel** [comment]
Released as [v3.0.0-beta4](https://github.com/petrleocompel/hikaxpro_hacs/releases/tag/v3.0.0-beta4)

Later it will be upstreamed to "master" and pushed to "stable" release of 3.0.0.
